### ISSUE TITLE
Update the image used in CircleCI

### ISSUE THREAD

**Issue by claudioandre** [magnumripper/JohnTheRipper#2386]
* Last year, Fedora 22 has reached its end of life for package update support including security fixes.
* The binary built (and zipped for distribution) is using buggy software. Example:

### System configuration (snip)
```
==Fedora 22 - x86_64 -  0% [                    ] ---  B/s |   0  B     --:-- ETA
 libpcap-devel       x86_64       14:1.7.3-1.fc22           updates       122 k
 openssl             x86_64       1:1.0.1k-15.fc22          updates       497 k
 openssl-devel       x86_64       1:1.0.1k-15.fc22          updates       1.3 M
 openssl-libs        i686         1:1.0.1k-15.fc22          updates       945 k
 openssl-libs        x86_64       1:1.0.1k-15.fc22          updates       943 k
```

**Comment by kholia** [comment]
Thanks for reporting this. I will try to update / replace this image with Fedora 25 on Monday.

**Comment by kholia** [comment]
This was fixed with PR #2395.
